JASP-Engine/JASP/R/correlation.R

```diff
@@ -409,8 +409,11 @@ Correlation <- function(jaspResults, dataset, options){
     result <- try(expr = {ppcor::pcor.test(x = x, y = y, z = z, method = method)}, silent = TRUE)
     if(isTryError(result)) {
       errors <- .extractErrorMessage(result)
+      if(startsWith(errors, "reciprocal condition number")) errors <- "System is computationally singular"
```

Comment:
What do you mean by "System" here? Don't you want to say something such as "Partial correlations cannot be computed: Covariance matrix of the controlling variables singular"

Reply:
Yeah good point... I mindlessly took the "System is computationally singular" because that's what `ppcor` package returns (Specifically, it returns: "System is computationally singular: reciprocal condition number = ..."). I changed this to "Partial correlation cannot be computed: covariance matrix is computationally singular."